### DISCUSSION TITLE
Failed to import theme

### DISCUSSION THREAD

**Discussion by mintnchocochip** [t1m0thyj/WinDynamicDesktop#430] [Q&A]
I get failed to import theme when trying to apply my custom theme. The error message is; " Failed to import 'theme' theme: Couldn't find image 3 at W113.jpg"
I checked if the files are of the same extensions, they are but I get this issue 
<img width="216" alt="image" src="https://user-images.githubusercontent.com/81812564/162970868-49db8b77-e981-40b0-bd22-8951e6262eaf.png">

**Comment by t1m0thyj** [comment]
Please upload the theme you're trying to import. Otherwise its hard to tell what's causing the error.
